zkevm-circuits/src/pi_circuit.rs

```diff
@@ -226,19 +226,19 @@ impl<F: Field> SubCircuitConfig<F> for PiCircuitConfig<F> {
         let tx_value = tx_table.value;
         let tag = tx_table.tag;
         let index = tx_table.index;
-        let tx_id_inv = meta.advice_column();
-        let tx_value_inv = meta.advice_column();
-        let tx_id_diff_inv = meta.advice_column();
+        let tx_id_inv = meta.advice_column_in(SecondPhase);
```

Comment:
This is marked as `SecondPhase` but we never allocate RLC-dependant values here. 
See: https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/5ab275565ddd35e7192623f131d0737784f04467/zkevm-circuits/src/pi_circuit.rs#L622L627 where always a Field Element is assigned. And it never carries RLC-related data

Reply:
Exactly. This is the expected usage indeed.